examples/cpp/dflags.h

```diff
@@ -85,24 +85,24 @@ class FlagRegistererBase {
 class FlagsRegistererSingleton {
  public:
   virtual ~FlagsRegistererSingleton() {}
-  virtual void RegisterFlag(const std::string& name, FlagRegistererBase* impl) = 0;
-  virtual void ParseFlags(int& argc, char**& argv) = 0;
-  virtual void PrintHelpAndExit(const std::map<std::string, FlagRegistererBase*>& flags) const {
+  virtual void RegisterFlag(const std::string & name, FlagRegistererBase * impl) = 0;
+  virtual void ParseFlags(int & argc, char **& argv) = 0;
+  virtual void PrintHelpAndExit(const std::map<std::string, FlagRegistererBase *> & flags) const {
     PrintHelp(flags, HelpPrinterOStream());
     std::exit(HelpPrinterReturnCode());
   }
 
  protected:
-  virtual void PrintHelp(const std::map<std::string, FlagRegistererBase*>& flags, std::ostream& os) const {
+  virtual void PrintHelp(const std::map<std::string, FlagRegistererBase *> & flags, std::ostream & os) const {
     os << flags.size() << " flags registered.\n";
     for (const auto cit : flags) {
-      os << "\t--" << cit.first << " , " << cit.second->TypeAsString() << "\n\t\t"
-         << cit.second->DescriptionAsString() << "\n\t\t"
+      os << "\t--" << cit.first << " , " << cit.second->TypeAsString() << "\n\t\t" << cit.second->DescriptionAsString()
```

Comment:
Regarding the change in `.clang-format`:

![line-too-long](https://cloud.githubusercontent.com/assets/2159447/8088302/27156586-0f55-11e5-8efe-43072d26430b.png)